### ISSUE TITLE
Command Line Tools?

### ISSUE THREAD

**Issue by SainteCelery** [sticklerm3/homebrew-pourhouse#18]
Thanks for setting this up! Do you plan to add a formula repo for EL's non-UI tools like Blowhole, alisma, cmpxat, etc?

**Comment by sticklerm3** [comment]
yeah, I've been looking into the particular stanzas on how to write them and so on.. basically this project has just been my practice to writing ruby formula for Homebrew. Since they're already binaries I've been reading on how to properly write them, any expertise is helpful and I'll have them up as soon as I have the time to work on them

**Comment by sticklerm3** [comment]
So I think I've figured out the way to properly write the formula! It seems to have gone mostly well in Travis! I'm still looking into making sure they install properly however so anyone who sees this could test alisma to start, that would be great! I know it downloads and the pkg installer in the caskroom, I'm just unable to verify whether the software properly installs or if it just places it there for you to install manually (like how little snitch works!). Once I'm sure that this is the correct formula, I'll add caveats stanza's with any information/instructions that may still be needed! Any input or feedback is always appreciated!

**Comment by sticklerm3** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Close issue #1](https://trello.com/c/mDPuF5vG/1-close-issue-1)

**Comment by WorkInProgress-Development** [comment]
I'm not exactly sure how to write this, but as alisma (maybe others) installs via a package file, users without administrator access cannot install it. I don't know whether there is a way to fix it, but when the alisma binary is downloaded from the eclectic light co website (eclecticlight.co), it comes in both a package and a binary, which can be moved elsewhere.

Apologies if this is already known and/or poorly written.
